### DISCUSSION TITLE
Não consigo fazer build para exportar EXE

### DISCUSSION THREAD

**Discussion by Anderson-Nascimento-Mendonca** [rustdesk/rustdesk#2690] [General]
Boa Tarde, extou tendo exporta para EXE mais quando faço o build, recebo essa mensagem e não gera o EXE.

![image](https://user-images.githubusercontent.com/85626023/210086806-ed3972b3-94d0-4108-a338-df3b5c06d9e7.png)

**Comment by edsonbandeira** [comment]
Conseguiu resolver ?
